common/persistence/dataManagerInterfaces_test.go

```diff
@@ -22,7 +22,9 @@ package persistence
 
 import (
 	"errors"
+	"fmt"
```

Comment:
let's rename the test file and corresponding go file to camel_case

Reply:
Renamed.